org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Antares/ImportAntaresTest.java

```diff
@@ -0,0 +1,70 @@
+package Debrief.ReaderWriter.Antares;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Enumeration;
+
+import Debrief.Wrappers.FixWrapper;
+import Debrief.Wrappers.Track.TrackSegment;
+import MWC.GUI.Editable;
+import MWC.GUI.Layer;
+import MWC.GUI.Layers;
+import junit.framework.TestCase;
+
+public class ImportAntaresTest extends TestCase {
+
```

Comment:
@saulhidalgoaular - I think you should also do some testing to verify the importer catches problems in the data (all those AntaresImportExceptions)